src/main/java/seedu/storage/Storage.java

```diff
@@ -0,0 +1,30 @@
+package seedu.storage;
+
+import seedu.classes.Ui;
+import seedu.type.IncomeList;
+import seedu.type.SpendingList;
+
+public class Storage {
+    private static IncomeList incomes;
+    private static SpendingList spendings;
+    private static int password;
+
+    public Storage(Ui ui) {
```

Comment:
Maybe could make the IncomeList, SpendingList and LoginStorage methods package private instead so only the storage package classes can access them